client/routes/namespace/index.vue

```diff
@@ -42,5 +50,11 @@ section.namespace {
   flex-direction: column;
   flex: 1 1 auto;
   overflow-y: auto;
+  position: relative
+}
+.bugreport {
+  position: absolute;
+  right: 1em;
+  top: 0.75em;
 }
```

Comment:
i dont like this solution but i wasnt sure how else to do this... my css isnt good enough! if you know a better solution pls be my guest